app/controllers/sweets_controller.rb

```diff
@@ -1,5 +1,26 @@
 class SweetsController < ApplicationController
+  before_action :set_sweet, only: [:show, :edit, :update, :destroy]
```

Comment:
Super nit pick, but I'd put an empty line below this.

Reply:
Also, you don't have an edit or destroy action so you shouldn't add them in the array.